scribble/app/views/home/index.html.erb

```diff
@@ -0,0 +1,10 @@
+<h1>RECENT POSTS</h1>
+<div class = "buttons">
+<a href="/posts">View All</a>
+  <%= link_to "Add New", new_post_path %>
+</div>
+<% @posts[0..2].each do |post| %>
+<div class = "post">
+      <h2><%=link_to "#{post.title}", post%></h2>
```

Comment:
No need for string interpolation here, since it's a simple string, it's not building a more complex string:

```ruby
<%= link_to post.title, post %>
```